src/logging/SOOSLogger.ts

```diff
@@ -45,7 +45,8 @@ class SOOSLogger {
   }
 
   setVerbose(verbose: boolean) {
-    this.verbose = verbose;
+    // TODO - PA-13294 - Consolidate verbose and log level to avoid overriding log level
```

Comment:
have to do this weird check because we set up log properties like this on integrations

```typescript
const args = this.parseArgs();
soosLogger.setMinLogLevel(args.logLevel);
soosLogger.setVerbose(args.verbose);
```